<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>721</width>
     <height>521</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Car ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Owner Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone No.</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addCarBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>640</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Car</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
